requirements.yml

```
@@ -1,2 +1,4 @@
 collections:
   - pulp.pulp_installer
+  - name: theforeman.operations
+    version: '>=0.2.0'
```

```suggestion
    version: '>=0.3.0'
```

to get the new variable names